sql/moz-fx-data-shared-prod/mozilla_org_derived/gclid_conversions_v1/query.sql

```diff
@@ -0,0 +1,72 @@
+WITH gclids_to_ga_ids AS (
+  SELECT
+    gclid,
+    ga_client_id,
+    stub_session_id,
+  FROM
+    mozilla_org_derived.gclid_conversions_v1
+  CROSS JOIN
+    UNNEST(all_reported_stub_session_ids) AS stub_session_id
+  WHERE
+    session_date >= DATE_SUB(@activity_date, INTERVAL @conversion_window DAY)
+    AND gclid IS NOT NULL
+),
+ga_ids_to_dl_token AS (
+  SELECT
+    ga_client_id,
+    stub_session_id,
+    dl_token,
+  FROM
+    mozila_org_derived.dl_ga_triplets_v1
+  WHERE
+    ga_client_id IS NOT NULL
+    AND stub_session_id IS NOT NULL
+),
+dl_token_to_telemetry_id AS (
+  SELECT
+    client_id AS telemetry_client_id,
+    first_seen_date,
+    attribution_dltoken AS dl_token,
+  FROM
+    telemetry_derived.clients_first_seen_v2
+),
+telemetry_id_to_activity AS (
+  SELECT
+    client_id AS telemetry_client_id,
+    search_count_all > 0 AS did_search,
+    ad_clicks_count_all > 0 AS did_click_ad,
+    TRUE AS was_active,
+  FROM
+    telemetry_derved.clients_daily_v6
+  WHERE
+    submission_date = @activity_date
```

Comment:
The idea here is that we want to filter based on _activity_ date: if an activity happens that we care about (i.e. a conversion event), then we find it here. We look back 30 days to see if that event has an associated GCLID - after 30 days, conversion events don't count! (that's the `conversion_window` parameter). This is standard practice for marketing campaigns.